reg_tests/CTestList.cmake

```diff
@@ -23,6 +23,13 @@ function(add_test_r_rst testname np)
     file(MAKE_DIRECTORY ${CMAKE_CURRENT_BINARY_DIR}/test_files/${testname})
 endfunction(add_test_r_rst)
 
+# Regression test with input
+function(add_test_r_inp testname np)
```

Comment:
I like the solution in NaluCFD/Nalu#462. Is there a reason to not rename `*_Input.i` to `*_rst.i`?

Reply:
Well, it's not a restart so I thought it was better to keep it named something separate.  Switching it to `_rst` will work too and it won't require the extra block in the CTestList file, but it's less clear what the test is doing.